offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/affiliation/search/SearchAffiliationView.groovy

```diff
@@ -83,7 +83,7 @@ class SearchAffiliationView extends FormLayout{
      */
     private void listenToAffiliationSelection() {
         this.affiliationGrid.addSelectionListener({
-            viewModel.selectedAffiliation = it.firstSelectedItem.orElse(null)
+            viewModel.selectedAffiliation = it.firstSelectedItem
```

Comment:
```suggestion
            viewModel.selectedAffiliation = it.firstSelectedItem.orElse(Optional.Empty)
```
maybe like this?

Reply:
no we have an optional in both cases now :)

Reply:
So this would assign an Affiliation or else an Optional<?> two different types of variables

Reply:
but then I would maybe just set it if the affiliation is present, I think otherwise there will be an error?

Reply:
```suggestion
            it.firstSelectedItem.isPresent? viewModel.selectedAffiliation = it.firstSelectedItem
```

Reply:
Why would there be an error? if an empty selection is present or something was deselected the Optional.empty of the selection event is propagated to the view model